src/components/GetStarted/Category/FeatureMedia/index.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { withStyles } from "@material-ui/styles";
+import Typography from "@material-ui/core/Typography";
+
+import { useStyles } from "./styles";
+
+const FeatureMedia = ({ classes, activeIndex }) => {
+  return (
+    <div>
+      {activeIndex === 0 && <TabContainer>Item One</TabContainer>}
+      {activeIndex === 1 && <TabContainer>Item Two</TabContainer>}
+      {activeIndex === 2 && <TabContainer>Item Three</TabContainer>}
+      {activeIndex === 3 && <TabContainer>Item Four</TabContainer>}
+      {activeIndex === 4 && <TabContainer>Item Five</TabContainer>}
+    </div>
+  );
+};
+
+function TabContainer(props) {
+  return (
+    <Typography component="div" style={{ padding: 24 }}>
```

Comment:
don't use inline styling. Use material-ui styles instead

Reply:
We're already using div in our App.. So either go with Typography or div but be consistent